src/XrdHeaders.cmake

```diff
@@ -39,6 +39,7 @@ set( XROOTD_PUBLIC_HEADERS
   XrdOuc/XrdOucIOVec.hh
   XrdOuc/XrdOucLock.hh
   XrdOuc/XrdOucName2Name.hh
+  XrdOuc/XrdOucPinLoader.hh
```

Comment:
Why does this need to be public?

Reply:
`XrdOucPinLoader.hh` is needed by `XrdClPlugInManager.hh`. We could, instead of adding this header, stop installing or installing privately both of them. I will check later if the header dependencies can be solved by installing less rather than more for this one.

Reply:
I made `XrdClPlugInManager.hh` private (no one could have used it before, because including it would cause a compilation error), and I install both `XrdClPlugInManager.hh` and `XrdOucPinLoader.hh` into `include/xrootd/private`. Later we can review which headers are currently exposed that don't make sense and simply remove them from the install target in a later version.